plugins/woocommerce/src/Blocks/Domain/Bootstrap.php

```diff
@@ -27,17 +28,22 @@
 use Automattic\WooCommerce\Blocks\Payments\Integrations\PayPal;
 use Automattic\WooCommerce\Blocks\Payments\PaymentMethodRegistry;
 use Automattic\WooCommerce\Blocks\Registry\Container;
+use Automattic\WooCommerce\Blocks\Templates\MiniCartTemplate;
 use Automattic\WooCommerce\Blocks\Templates\CartTemplate;
-use Automattic\WooCommerce\Blocks\Templates\CheckoutHeaderTemplate;
 use Automattic\WooCommerce\Blocks\Templates\CheckoutTemplate;
-use Automattic\WooCommerce\Blocks\Templates\ClassicTemplatesCompatibility;
+use Automattic\WooCommerce\Blocks\Templates\CheckoutHeaderTemplate;
 use Automattic\WooCommerce\Blocks\Templates\OrderConfirmationTemplate;
+use Automattic\WooCommerce\Blocks\Templates\ClassicTemplatesCompatibility;
 use Automattic\WooCommerce\Blocks\Templates\ProductAttributeTemplate;
+use Automattic\WooCommerce\Blocks\Templates\ProductCatalogTemplate;
+use Automattic\WooCommerce\Blocks\Templates\ProductCategoryTemplate;
+use Automattic\WooCommerce\Blocks\Templates\ProductTagTemplate;
```

Comment:
I know that some templates have been registered in this `Bootstrap` class before but I think `BlockTemplatesRegistry` is the better place to register and store registered templates. This PR is already complex so I'm not suggesting we update anything but want to raise this point for discussion.

Reply:
Good idea, I changed how this works in 48800e5c55ee2d74f2515eb502cfce8d6dfc9e3e. Instead of registering template classes in the container, I create instances of them directly in `BlockTemplatesRegistry`.